src/app/components/course-island/course-island.component.html

```diff
@@ -17,6 +17,14 @@ <h2 class="tui-island__title">{{ course.name }}</h2>
         <tui-tag
             [status]="course.is_registered ? 'success' : 'warning'"
             [value]="course.is_registered ? 'Registered' : 'Not Registered'"
+            class="tui-space_right-2"
+            size="m"
+        ></tui-tag>
+        <tui-tag
+            *ngIf="course.instructor === user?.id"
+            [status]="course.instructor === user?.id ? 'custom' : 'warning'"
```

Comment:
The component is not visible when unless `course.instructor === user?.id` so `[status]="course.instructor === user?.id ? 'custom' : 'warning'"` doesn't makes sense.